website/index.html

```diff
@@ -1,11 +1,22 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="manifest" href="static/manifest.json">
+
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0 shrink-to-fit=no">
```

Comment:
I think this should be `width=device-width,initial-scale=1.0,shrink-to-fit=no` (use `,` to separate instead of space)